google-cloud-pubsub/lib/google/cloud/pubsub/received_message.rb

```diff
@@ -170,7 +170,9 @@ def published_at
         #   subscriber = sub.listen do |received_message|
         #     puts received_message.message.data
         #
-        #     received_message.acknowledge!
+        #     received_message.acknowledge! do |result|
+        #         puts result.status
```

Comment:
Please use the correct indentation for all sample code. (There are additional cases below.)